### DISCUSSION TITLE
FOD distances

### DISCUSSION THREAD

**Discussion by peraltajuan** [FLOSIC/PublicRelease_2020#14] [Q&A] [answered]
Hello, I have a question bugging me. What is the minimum distance between optimized FODs for the n=1,2 shells of a typical transition metal? or anything heavier?

**Comment by jacks1ka** [comment, marked as answer]
For Cr, it looks like the n = 1 to n = 2 separation is around 0.2 Bohr.  Similar for Fe.  You can find optimized FOD positions for the isolated atoms in the supplemental material of "Self-consistent self-interaction corrected density functional theory calculations for atoms using Fermi-Löwdin orbitals: Optimized Fermi-orbital descriptors for Li – Kr."  D-y. Kao, K. Withanage, T. Hahn, J. Batool, J. Kortus, and K. Jackson, J. Chem. Phys. 147, 164107 (2017).  doi: 10.1063/1.4996498
